Makefile

```diff
@@ -4,6 +4,7 @@ help:
 	@echo 'make services          Run the services that `make dev` requires'
 	@echo "                       (Postgres) in Docker"
 	@echo 'make db                Upgrade the DB schema to the latest version'
+	@echo 'make alembic           Run alembic (DB migrations) commands. Default to `history`
```

Comment:
Added the new alembic target, left the other one unchanged in case someone has muscle memory to use that for alembic stuff.